docs/iris/gallery_code/meteorology/plot_lagged_ensemble.py

```diff
@@ -80,83 +84,68 @@ def main():
         right=0.925,
     )
 
-    # iterate over all possible latitude longitude slices
+    # Iterate over all possible latitude longitude slices.
     for cube in last_timestep.slices(["latitude", "longitude"]):
 
-        # get the ensemble member number from the ensemble coordinate
+        # Get the ensemble member number from the ensemble coordinate.
         ens_member = cube.coord("realization").points[0]
 
-        # plot the data in a 4x4 grid, with each plot's position in the grid
-        # being determined by ensemble member number the special case for the
-        # 13th ensemble member is to have the plot at the bottom right
+        # Plot the data in a 4x4 grid, with each plot's position in the grid
+        # being determined by ensemble member number.  The special case for the
+        # 13th ensemble member is to have the plot at the bottom right.
         if ens_member == 13:
             plt.subplot(4, 4, 16)
         else:
             plt.subplot(4, 4, ens_member + 1)
 
-        cf = iplt.contourf(cube, contour_levels)
+        # Plot with 50 evenly spaced contour levels (49 intervals).
+        cf = iplt.contourf(cube, 49, vmin=data_min, vmax=data_max)
 
-        # add coastlines
+        # Add coastlines.
         plt.gca().coastlines()
 
-    # make an axes to put the shared colorbar in
+    # Make an axes to put the shared colorbar in.
     colorbar_axes = plt.gcf().add_axes([0.35, 0.1, 0.3, 0.05])
     colorbar = plt.colorbar(cf, colorbar_axes, orientation="horizontal")
-    colorbar.set_label("%s" % last_timestep.units)
-
-    # limit the colorbar to 8 tick marks
-    import matplotlib.ticker
+    colorbar.set_label(last_timestep.units)
 
+    # Limit the colorbar to 8 tick marks.
     colorbar.locator = matplotlib.ticker.MaxNLocator(8)
     colorbar.update_ticks()
 
-    # get the time for the entire plot
-    time_coord = last_timestep.coord("time")
-    time = time_coord.units.num2date(time_coord.bounds[0, 0])
+    # Get the time for the entire plot.
+    time = last_time_coord.units.num2date(last_time_coord.bounds[0, 0])
 
-    # set a global title for the postage stamps with the date formated by
-    # "monthname year"
-    plt.suptitle(
-        "Surface temperature ensemble forecasts for %s"
-        % (time.strftime("%B %Y"),)
-    )
+    # Set a global title for the postage stamps with the date formated by
+    # "monthname year".
+    time_string = time.strftime("%B %Y")
+    plt.suptitle(f"Surface temperature ensemble forecasts for {time_string}")
```

Comment:
I thought it might make it more obvious what's going on if the `time_string` is calculated before creating the f-string.